web/package.json

```diff
@@ -54,19 +56,19 @@
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "eslint-plugin-vue": "^9.20.1",
     "eslint-plugin-vue-scoped-css": "^2.7.2",
+    "postcss": "^8.4.33",
     "prettier": "^3.2.4",
     "replace-in-file": "^7.1.0",
+    "tailwindcss": "^3.4.1",
```

Comment:
tailwindcss should be moved to dependencies

Reply:
postcss and autoprefixer too, assuming you keep build dependencies in `dependencies`.